CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/) and this project adheres to [Semantic Versioning](https://semver.org/).
 
+## [Unreleased] - 2023-05-11
+### Fixed
+- Remove exit() after sending the response and use fastcgi_finish_request() instead
```

Comment:
```suggestion
## [Unreleased]
### Fixed
- restored compatibility with the Shopware market place review & release process
```

Short Shopgate changelog guideline because you've never done this before:
* You don't need to add a date, instead just add your stuff under "Unreleased". We'll make a new title and date when publishing a new version.
* Try keeping the changelog non-technical as we expect the "audience" for this to be merchants or agencies that are trying to see if their issue has been resolved, not so much how it has been done.

Reply:
Great, thanks for the info.